src/components/Layout/index.tsx

```diff
@@ -37,50 +38,33 @@ const Layout: FC<Props> = ({ children }) => {
             {isLoading && <Loader />}
             <div className={styles.wrapper}>
                 <div className={styles.header}>
-                    {navBarContent('Tasks', '/', router.pathname === '/')}|
-                    {navBarContent(
-                        'Issues',
-                        '/issues',
-                        router.pathname === '/issues'
-                    )}
-                    |
-                    {navBarContent(
-                        'Mine',
-                        '/mine',
-                        router.pathname === '/mine'
-                    )}
-                    {/* TODO: Uncomment when DS(Chanllenges) is ready */}
-                    {/* |
-                    {navBarContent(
-                        'DS',
-                        '/challenges',
-                        router.pathname === '/challenges'
-                    )} */}
-                    |
-                    {navBarContent(
-                        'Open PRs',
-                        '/openPRs',
-                        router.pathname === '/openPRs'
-                    )}
-                    |
-                    {navBarContent(
-                        'Stale PRs',
-                        '/stale-pr',
-                        router.pathname === '/stale-pr'
-                    )}
-                    |
-                    {navBarContent(
-                        'Idle Users',
-                        '/idle-users',
-                        router.pathname === '/idle-users'
-                    )}
-                    {dev && (
-                        <>
-                            |
+                    {navBarContentMock.map((element, index) => (
+                        <React.Fragment key={index}>
                             {navBarContent(
-                                'Availability Panel',
-                                '/availability-panel'
+                                element.title,
+                                element.refURL,
+                                router.pathname === element.pathName
                             )}
+                            {element.pipeSymbol}
+                        </React.Fragment>
```

Comment:
Had some query here...
If we are importing import React, { FC, ReactNode } from 'react';
Can't we also import Fragment here?
If you want to pass key to a Fragment, you can’t use the <>...</> syntax. You have to explicitly import Fragment from 'react' and render <Fragment key={yourKey}>...</Fragment>

Reply:
fixed